compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/ConfigUtils.cpp

```diff
@@ -86,6 +86,8 @@ LogicalResult setMatmulLoweringConfig(IREE::GPU::TargetAttr target,
     return failure();
 
   GPUMMAHeuristicSeeds seeds;
+  int64_t inBitWidth = lhsElemType.getIntOrFloatBitWidth();
+  int64_t cacheLineSizeBits = 1024;
```

Comment:
nit: make this a constant `constexpr int64_t kCacheLineSizeBits = 128 * 8;` and put towards the top of this namespace